### ISSUE TITLE
Item Pickup Manager does not work between dimensions

### ISSUE THREAD

**Issue by dhalek** [maruohon/enderutilities#76]
I'm using MineCraft 1.12.2 and Ender Utilities 1.12.2-0.7.11.
When I'm in the Nether and my Item Pickup Manager is linked to a chest in the overworld, an I put an Item into the transfer slot it just stays there an is not transferred. The capacitator is fully charged. I also tested The End and a RFTools dimension. I have an Energy Bridge Transmitter in the End and a Receiver in the Overworld. The pickup manager has one link crystal installed which is linked to a chest in the Overworld and the transport works onliy local to this dimension.

I believe that this worked interdimensionally in some earlier versions.

Do I miss something here or is that really a bug?

**Comment by maruohon** [comment]
It is meant to be working for sure, I'll have to look into it. Is there anything special about the target inventory? Or is it just a vanilla chest for example?

**Comment by dhalek** [comment]
Yes it is just a vanilla chest. Well that is connected via Makanism pipes to a ProjectE condenser.
But the transport works if I'm in the overworld so it's not completely broken.
I will test, if the transport works in the Nether locally and report bank.

**Comment by maruohon** [comment]
How close to the target were you in these cases, ie. was the target Chest within the player's loaded chunks or outside them?

**Comment by dhalek** [comment]
Well, my first tests were quite close to the target. I just tested it from about 500 blocks away and it worked. My base is chunkloaded with PersisentBits-Chunkloader.
Hope that helps.

**Comment by dhalek** [comment]
I just loaded an older world and there it works. This one had EnderUtilities 0.7.8. It stopped working after I updated to 0.7.11.
So there was a bug introduced after 07.8.
Hope that helps.

**Comment by maruohon** [comment]
Okay I found the issue. In 0.7.9 I had a commit for cleaning up some of the code in the IPM item, and I switched into using an existing util method for getting the bound inventory. But I didn't notice that the util method also had the check for the Advanced Ender Core in it, since it was previously only used in the Ender Tool and Sword items... And since the IPM doesn't use/have that module, the cross-dimensional bound inventory access always failed due to that check.
